nussknacker-dist/src/universal/conf/dev-oauth2-application.conf

```diff
@@ -0,0 +1,129 @@
+# This config is exposed in development dist and locally in development (se ui/server/runServer.sh)
```

Comment:
Do we need yet another file here? Can we use dev-application and use env variables?